notebooks/image_models/solutions/serving.ipynb

```diff
@@ -0,0 +1,1639 @@
+{
```

Comment:
<ul><li> (i.e., tf.Variables) and computation --&gt;  (i.e., tf.Variables) and the computation graph</li><li>It does not require the original model building code to run --&gt; We don't need the original model building code to run a model exported with SavedModel</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Thank you. Updated.</p>